src/main/resources/custom-events.sk

```diff
@@ -0,0 +1,36 @@
+
+# This file allows you to register custom events that aren't already registered under Skript.
+# The event classes you want to register must exist in the server class loader. So events from Paper/Spigot.
+# If you want to register an event from a plugin API, you must edit the plugin.yml in the jar of Skript to include:
+#
+# softdepend: [YourPlugin, SQLibrary, Vault, WorldGuard, Residence, PreciousStones, GriefPrevention]
+#
+# Reminder to maintain any existing defined plugins in this array as this example should not be directly copied.
+#! Additions or changes to this file requires a server restart.
+
+#! Internal version of this configuration. Don't modify.
+version: 1
+
+# The format must include a section where the section name is the event name.
+# The event section must include:
+# 	description - describing the event
+#	class - the class name to search for
+# 	pattern - the syntax pattern of the event
+# The optional pattern [on] gets prefixed on every Skript event.
+# Pattern cannot have expressions inside currently.
+# All event-values will be automatically applied.
```

Comment:
This is only partially true
Event values for parent events will be added, but there's no support for event values specific to the event. For example, with the campfire cook event, there's no way to get [getTotalCookTime](https://jd.papermc.io/paper/1.19/org/bukkit/event/block/CampfireStartEvent.html#getTotalCookTime())(). Perhaps this could be reworded to be a bit clearer.